lib/block-supports/layout.php

```diff
@@ -76,14 +76,22 @@ function gutenberg_get_layout_style( $selector, $layout, $has_block_gap_support
 			'space-between' => 'space-between',
 		);
 
+		$flex_wrap_options = array(
+			'wrap'   => 'wrap',
+			'nowrap' => 'nowrap',
+		);
+		$flex_wrap         = ! empty( $layout['flexWrap'] ) && array_key_exists( $layout['flexWrap'], $flex_wrap_options ) ?
+			$flex_wrap_options[ $layout['flexWrap'] ] :
+			'wrap';
+
```

Comment:
```suggestion
		$flex_wrap_options = array( 'wrap', 'nowrap' );
		$flex_wrap         = ! empty( $layout['flexWrap'] ) && in_array( $layout['flexWrap'], $flex_wrap_options, true ) ?
			$layout['flexWrap'] :
			'wrap';
```

Key/value pairs are the same. So we can omit keys and use the options array for validation.

Reply:
I actually used an object at first because in some cases (like justification) is better to map a key to some values, but in this case since it we will only have this two values, your suggestion makes more sense. I updated, thanks!